File: articles/feedback-loops-in-algorithmic-systems/typescript/feedback_risk.ts
const amplification = 0.82;
const concentration = 0.76;
const intervention = 0.44;
const drift = 0.28;
const recursiveData = 0.31;
const score = (amplification + concentration + intervention + drift + recursiveData) / 5.0;
console.log(`feedback_risk_score=${score.toFixed(4)}`);

File: articles/feedback-loops-in-algorithmic-systems/rust/feedback_risk.rs
fn main() {
    let amplification = 0.82;
    let concentration = 0.76;
    let intervention = 0.44;
    let drift = 0.28;
    let recursive_data = 0.31;
    let score = (amplification + concentration + intervention + drift + recursive_data) / 5.0;
    println!("feedback_risk_score={:.4}", score);
}
